cadcutils/cadcutils/net/ws.py

```diff
@@ -100,13 +100,14 @@
 class BaseWsClient(object):
     """Web Service client primarily for CADC services"""
 
-    def __init__(self, resource_id, agent, anon=True, cert_file=None, retry=True, host=None):
+    def __init__(self, resource_id, subject, agent, retry=True, host=None):
         """
         Client constructor
         :param anon  -- anonymous access or not. If not anonymous and
```

Comment:
Nope. All the authentication info is now included in the Subject. So for an anon client you have to pass Subject() as argument.